roles/agg_pushgateway_exporter/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+prom_aggregation_gateway_install_dir: "/opt/prometheus-exporters/prom-aggregation-gateway"
+prom_aggregation_gateway_binary: "{{ prom_aggregation_gateway_install_dir }}/prom-aggregation-gateway"
+prom_aggregation_gateway_install_url: "github.com/LarkTechnologies/prom-aggregation-gateway"
```

Comment:
you'll need to use fork and branch: https://github.com/dimagi/prom-aggregation-gateway/tree/sk/guage-aggregation

Reply:
Changed.